### DISCUSSION TITLE
Private 1:1 OTC trades executed on Bisq

### DISCUSSION THREAD

**Discussion by Piste** [bisq-network/bisq#5314] [Ideas]
Let's say I found a party that has the other side of my trade. I don't know them but I know they would fill my trade. I don't want to bother myself with evaluating my level of trust with them. It sounds like Bisq would be ideal for this: we both go on Bisq, and leave the trust component to Bisq's logic. But I want to trade with this party only because I committed to the trade. Hence I want to put up a trade privately with another party. I don't want my bid or ask to be publicly displayed or hit. Can this be done?

## Benefits
- Simple trades: just need a conterparty ID (could be a Bisq? wallet address), a currency pair. Fill each other at market price
- Private, like Bisq is 
- Trustless: I don't have to worry about whether this person is trying to scare me or not. The trade will or will not happen, simple
- More biz for Bisq: each trade compensates infrastructure maintainers
- Bisq continues to be the ledger for these trades 

## Implementation
(I have no idea what I'm talking about)
It looks to me like most or all of the code and design are already here: It's really just posting a trade but not advertising it. Then anyone who's posted a trade can then send a permalink or contract address to this trade offer to their party and say "hit me here".

**Comment by chimp1984** [comment]
The offer code contains some fields for that use case but it was never implemented in the UI. The demand for that use case seems rather low, so no dev efforts went into it so far.

**Reply by Conza88** [reply]
Lower trading fee's for it somehow?

**Comment by pazza83** [comment]
@Piste thanks for the post. 

I think this is a fantastic idea. The use case is great.

Also to move it further for another use case:

_Let's say I found a person selling a anonymous BTC gambling site. I don't know them, or trust them, but I would buy their site from them. I don't want to bother myself with evaluating my level of trust with them. We both go on Bisq, and leave the trust component to Bisq's logic. But I want to trade with this party only because I committed to the trade. Hence I want to put up a trade privately with another party. I don't want my trade to be publicly displayed, nor do I want to rely on a trusted third party, or deal with KYC or a traditional escrow service provide._

I mentioned this thread here: https://github.com/bisq-network/bisq/discussions/5285#discussioncomment-480148

I think it has potential for private 1 to 1 trades for anything using BTC as collateral (ability to mediate permitting).

**Reply by Conza88** [reply]
> I think it has potential for private 1 to 1 trades for anything using BTC as collateral (ability to mediate permitting).

Especially: [https://github.com/bisq-network/bisq/issues/5044](https://github.com/bisq-network/bisq/issues/5044) hehe. From an efficiency POV you see what they're communicated to get context immediately. You see who has tried to resolve, and who hasn't... or what core issues are. Mediator gets up to speed swiftly, and can propose next steps.

**Comment by RififiCastorjunior** [comment]
how will you avoid being in there https://bisq.network/markets/ ? Bisq is an open project so growth team need those data.

**Comment by wallclockbuilder** [comment]
Just have your counterparty create and post a normal bisq `make offer`(which is still anonymous). 
Then use a unique exchange rate, which is how you find his anonymous offer(BTC for sale at $57.777).
You get what you want: private trade & security(insurance).

**Reply by Piste** [reply]
Would your solution, @wallclockbuilder , prevent the trade from being seen and potentially taken by another party? 
What I originally had in mind is: keep the trade private to both parties until it's done. Once done, you can add it to any public ledger.

**Reply by wallclockbuilder** [reply]
Nope

**Reply by RififiCastorjunior** [reply]
I wonder why use Bisq, why not going to a central trading plateform ?

**Reply by Conza88** [reply]
Because there's KYC?

**Reply by RififiCastorjunior** [reply]
> Simple trades: just need a conterparty ID (could be a Bisq? wallet address), a currency pair. Fill each other at market price

So Bisq will use ID for private trades. That sound familiar no ?

**Reply by ripcurlx** [reply]
You can always exchange and use the onion address to find a trading peer. One way to prevent someone of taking an offer would be to create an offer with a `specific bank` account and use a bank that doesn't exist. Doing it like that will make sure that no-one else is taking this offer. The drawback is that you need to exchange the trade details (payment method account details) afterwards through the trader chat.

**Reply by Piste** [reply]
The trades I had in mind for this use case were crypto to crypto

**Comment by RififiCastorjunior** [comment]
> Let's say I found a party that has the other side of my trade. I don't know them but I know they would fill my trade. 

I wonder why the point of using Bisq, you could just use a multisig and do your trade.

**Reply by MwithM** [reply]
You would need to use a 2of2 multisig, Mutual Assured Destruction and an atomic transaction to send funds there. There's no easy tool to do that yet.
Also, in case your peer disappears, or keys are lost, funds would be lost. And there's a possibility of blackmail from the part who loses less in a Mutual Assured Destruction scheme as loses aren't symmetric.
